### ISSUE TITLE
Feature request: hccapx file option

### ISSUE THREAD

**Issue by bluegizmo83** [derv82/wifite2#178]
It would be nice to implement and option for WPA captures to output to .hccapx files instead of .cap files. This would save some time in converting them afterwards for hashcat.

**Comment by ghost** [comment]
yes, and option to write directly in JTR format too using wpapcap2jhon , for both PMKID and Handshake
